File: joguito.py
import random
pontos = 0
while 1:
	a = random.randrange(0, 100);
	print("o numero a seguir vai ser maior que 50?")
	print("S para sim ou N para não")
	print("Pressione F para sair")
	n = input()
	if n == "F":
  		break
	if n == 'S' and a >= 50:
		pontos = pontos + 1
		print("ACERTOU!")
	elif n == 'S' and a < 50:
		print("ERROOUU!!")
	elif n == 'N' and a < 50:
		pontos = pontos + 1
		print("ACERTOU!")
	elif n == 'N' and a > 50:
		print("ERROOUU!!")
	if n == "F":
  		break
	print("valor sorteado:",a)
print("pontos:",pontos)

File: js_version/Joguito.js
function getRandomInt(min, max) {
    min = Math.ceil(min);
    max = Math.floor(max);
    return Math.floor(Math.random() * (max - min)) + min;
}
var pontos = 0,
    a;
var verificador;

while (verificador !== 'F') {
    console.log("o proximo numero ira ser maior que 50?");
    console.log("S para Sim e N para Não.");
    verificador = prompt("");
    a = getRandomInt(0, 100);
    if (verificador === 'S' && a > 50) {
        console.log("ACERTOU!");
        pontos++;
    } else if (verificador === 'S' && a < 50) {
        console.log("ERROOUU!!");
    }
    if (verificador === 'N' && a > 50) {
        console.log("ERROOUU!!");
    } else if (verificador === 'N' && a < 50) {
        console.log("ACERTOU!");
        pontos++;
    }
    console.log(a);
}
console.log("pontos", pontos);